src/header/internal.tsx

```diff
@@ -60,7 +64,11 @@ export default function InternalHeader({
             isRefresh && styles[`title-variant-${variantOverride}-refresh`]
           )}
         >
-          <HeadingTag className={clsx(styles.heading, styles[`heading-variant-${variantOverride}`])}>
+          <HeadingTag
```

Comment:
Would it be possible to avoid modifying our header component? You can maybe create some `src/header/focusable-header.tsx` and see how much code is possible to reuse between that and `src/header/internal.tsx`.

Reply:
+1

This header will never have actions or counter prop. Can be simplified to `<Box variant="h2">` with some additions